### ISSUE TITLE
Styles seem a little odd

### ISSUE THREAD

**Issue by bqbbo** [Naezr/ShyFox#173]
![image](https://github.com/user-attachments/assets/39552c78-11cd-4977-a762-0718c065c709)

I'm not sure if I'm really stupid and missed something important, but I can't seem to get this to replicate what I've seen in the readme. Does anyone have any suggestions?

**Comment by KAGEYAM4** [comment]
Did you imported Sideberyy config?  If no then do it. Read the readme again to check if you didn't miss anything else.

Otherwise try going into sidebery settings and change `Color scheme` to `firefox` ->
![image](https://github.com/user-attachments/assets/013bd0f0-3d45-48ee-bd89-0a2ccbb18ccd)

**Comment by dacoder101** [comment]
> Did you imported Sideberyy config? If no then do it. Read the readme again to check if you didn't miss anything else.
> 
> Otherwise try going into sidebery settings and change `Color scheme` to `firefox` -> ![image](https://private-user-images.githubusercontent.com/75798544/373067866-013bd0f0-3d45-48ee-bd89-0a2ccbb18ccd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5MjM3NDMsIm5iZiI6MTcyNzkyMzQ0MywicGF0aCI6Ii83NTc5ODU0NC8zNzMwNjc4NjYtMDEzYmQwZjAtM2Q0NS00OGVlLWJkODktMGEyY2NiYjE4Y2NkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDAzVDAyNDQwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU2MTg4NGYwODNhMzZlNWNkNzk2NGNkNjExYjI2ZmU0NzRmYTJhZDVlMWI0MDE2ZGQwZjk1MGRhMWU3OGJhN2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.W6285YdnCrXK7GUP3fp8TJIZHWFK2IgwUtC89q38y38)

Yep, everything seems to be in order. I've reread the README installation process a lot and I don't think I've missed anything.

**Comment by KAGEYAM4** [comment]
Follow these -->
1. Remove the addon color theme ( purple ) you are using.
2. select and enable  Dark theme ( inbluilt )
3. Apply the above setting for sidebery from previous message.



Now for coloring .. follow this - https://github.com/Naezr/ShyFox?tab=readme-ov-file#customization

**Comment by dacoder101** [comment]
> Follow these -->
> 
>     1. Remove the addon color theme ( purple ) you are using.
> 
>     2. select and enable  Dark theme ( inbluilt )
> 
>     3. Apply the above setting for sidebery from previous message.
> 
> 
> Now for coloring .. follow this - https://github.com/Naezr/ShyFox?tab=readme-ov-file#customization

Thanks for the help; for some reason, automatic coloring was on despite me thinking it was turned on dark.
![image](https://github.com/user-attachments/assets/8989f9ca-8628-4c6b-949c-4c9cba35470c)
I'm unsure if any of the showcased UI is out of the box; right now it looks like the above.

**Comment by bqbbo** [comment]
I'm a little naive and it probably has something to do with whatever WM Windows uses. I will install this on arch and see if I can replicate something similar to the showcased UI.

Edit: window managers matter
